src/components/Modal/NewAppsToAuthorizeModal/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useCallback, useState } from "react"
+import { useNavigate } from "react-router-dom"
+import InfoBox from "../../InfoBox"
+import {
+  BodyLg,
+  Button,
+  Card,
+  H5,
+  LabelMd,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  Radio,
+  RadioGroup,
+  Stack,
+} from "@threshold-network/components"
+import { BaseModalProps } from "../../../types"
+import withBaseModal from "../withBaseModal"
+import { useStakingState } from "../../../hooks/useStakingState"
+import { getStakeType } from "../../../utils/getStakeType"
+
+const NewAppsToAuthorizeModal: FC<BaseModalProps> = ({ closeModal }) => {
+  const { stakes } = useStakingState()
+
+  const [selectedAuthorizeAddress, setSelectedAuthorizeAddress] = useState("")
+
+  const navigate = useNavigate()
+
+  const routeToStake = useCallback(() => {
+    if (selectedAuthorizeAddress.length > 0) {
```

Comment:
Same as above I'm pretty sure this can be simplified to:

```suggestion
    if (selectedAuthorizeAddress) {
```